index.js

```diff
@@ -50,15 +50,16 @@ module.exports = function (sails) {
 
         initConnections: function () {
             var connections = {}, connection, connectionName;
+            var datastore = sails.config.models.datastore || 'default';
 
```

Comment:
Hmm.... Why not to define a key for searching connection properties, like
```
var connectionKey = sails.config.models.datastore ? 'datastore' : 'connection';
```

Reply:
I've updated it already to support both variants of sails. Please have a look.